config/packages/validator.yaml

```diff
@@ -0,0 +1,8 @@
+framework:
```

Comment:
This file has created with composer, but not sure it's needed for now.

Reply:
This file is an automatic configuration of the validator package we can leave it additionally it is a best practice for the validation of the email